### PR TITLE
Updated docstrings to be consistent with course notes + due dates

### PR DESCRIPTION
**Pull request by justinl1996**
I've updated the docstrings in the tutorial exercises to be consistent with the new style guide adopted in 2017. I've also fixed the interpreter window and the link to the visualizer (same as the course webpage)

### CODE REVIEW

File: problem_db/class_point.tut/preload.py [hunk 1]
```diff
@@ -3,9 +3,18 @@
 
 
 class Point(object):
+    """A 2D point in the cartesian plane"""
     def __init__(self, x, y):
+        """
+        Constructs a point object given the x and y coordinates
```
Comment (jgat) [inline problem_db/class_point.tut/preload.py]: Most places in these files write the function comment in the imperative mood (e.g. "Return ...", "Increment ..."), not the indicative mood (e.g. "Returns ...", "Increments ...").

I suggest choosing one and being consistent throughout.

In other words, should this read "Constructs ..." or "Construct ..."?

File: problem_db/if_vowel.tut/preload.py [hunk 2]
```diff
@@ -1,4 +1,13 @@
 def is_vowel(c):
+    """Returns True iff 'c' is a vowel.
```
Comment (jgat) [inline problem_db/if_vowel.tut/preload.py]: Same here, should this be "Returns" or "Return"?

File: problem_db/inherit_poly.tut/preload.py [hunk 3]
```diff
@@ -4,11 +4,17 @@ class Shape():
 
     """
     def __init__(self, origin=(0, 0)):
+        """Constructs a shape object
```
Comment (jgat) [inline problem_db/inherit_poly.tut/preload.py]: and here.

File: problem_db/inherit_poly.tut/preload.py [hunk 4]
```diff
@@ -18,25 +24,43 @@ def vertices(self):
...
-
         """
         raise NotImplementedError()
 
 
 class Square(Shape):
+    """A Square object"""
     def __init__(self, side_length, origin=(0, 0)):
+        """
+        Constructs a square object
```
Comment (jgat) [inline problem_db/inherit_poly.tut/preload.py]: and here.

File: problem_db/class_point.tut/preload.py (hunk 1 above)
Comment (justinl1996) [inline problem_db/class_point.tut/preload.py]: Good point I should be consistent with this. I'll stick with imperative mood. Thanks for the feedback :), I still recall being tutored by you in 2014 SM2....

**Pull request merged**